### PR TITLE
allow solo5 0.8 line (works fine with the current set of things)

### PR DESCRIPTION
**Pull request by palainp**
Authored-by: @hannesm hannes@mehnert.org

### CODE REVIEW

**Review by dinosaure — APPROVED** [review]

**Comment by hannesm** [comment]
thanks, a line in CHANGES would be appreciated :)

**Comment by palainp** [comment]
Oops, sorry, this is done now :)

File: CHANGES.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
   (* mirage >= 2.3.4 & < 5.0.0 *) (#1466 @hannesm, fixes #1381)
 - Use \033 instead of \e in generated Makefile (#1464 @hannesm)
 - Adapt to mirage-logs 2.0.0 (#1460 @hannesm)
+- Allow version 0.8.0 of solo5 (#1490 @hannesm @palainp)
```
Comment (hannesm) [inline CHANGES.md:16]:
```suggestion
- Allow version 0.8.0 of solo5 (#1432 #1490 @hannesm @palainp) 
```

**Comment by hannesm** [comment]
thanks, I added the earlier PR as well :)

**Pull request merged**